packages/fx-core/src/plugins/resource/bot/utils/telemetry-helper.ts

```diff
@@ -15,6 +15,10 @@ export class telemetryHelper {
   ): void {
     properties[TelemetryKeys.Component] = PluginBot.PLUGIN_NAME;
     properties[TelemetryKeys.Success] = TelemetryValues.Success;
+    properties[TelemetryKeys.AppId] =
```

Comment:
we moved the common code in to a function. please follow us.

Reply:
https://github.com/OfficeDev/TeamsFx/pull/1197

Reply:
ok